h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonReaderDriver.java

```diff
@@ -551,23 +551,15 @@ private void parseMultiPolygonMetadata(JsonParser jp) throws IOException, SQLExc
             jp.nextToken(); // START_ARRAY [ coordinates             
             jp.nextToken(); //Start the polygon
             while (jp.getCurrentToken() != JsonToken.END_ARRAY) {
-                //Parse the polygon
+                //Parses the polygon
                 jp.nextToken(); //Start the RING
                 int linesIndex = 0;
                 while (jp.getCurrentToken() != JsonToken.END_ARRAY) {
-                    if (linesIndex == 0) {
```

Comment:
Illogical alternative: if linesIndex == 0 or linesIndex != 0, we do the same thing!